build.yml

```
@@ -160,6 +160,7 @@ docker-targets:
     args:
       from: ubuntu:bionic
       jpeg: libjpeg-turbo8-dev
+      python_package: python3
```

```suggestion
      python: python3
```
Does just having `python3` work, or you need `python-is-python3` for it to work?

---

Perfect, thanks!